lib/semian/resource.rb

```diff
@@ -5,7 +5,7 @@ class Resource #:nodoc:
     class << Semian::Resource
       # Ensure that there can only be one resource of a given type
       def instance(*args)
-        Semian.resources[args.first] ||= ProtectedResource.new(args.first, new(*args), nil)
+        Semian.resources[args.first] ||= ProtectedResource.new(args.first, new(args.first, **args[1]), nil)
```

Comment:
I'm unsure if there's a better way to deal with this

Reply:
Oh, I missed that one as well. 

Same thing, `*args, **kwargs` is the way to go, unless I'm missing something.

Reply:
Nope, i didn't know you could do that, switched both over